Codeinwp/feedzy-rss-feeds
# Add feature usage observer

## Summary 

- Enabled the SDK telemetry for this product.
- The plugin settings are tracked via the Form submit event.
- Added block options tracking.

Also added the toggle for Telemetry since WP does not want telemetry by default.

![image](https://github.com/Codeinwp/feedzy-rss-feeds/assets/17597852/064d485a-567f-4ce5-8724-42e26a50eafc)


## Testing

- Enable Telemetry in the settings.
- Check if something is broken.
- Post the test instance URL so we can check in the metabase for the saved events.
